base/deprecated.jl

```diff
@@ -1510,6 +1510,12 @@ end
 # Issue #26248
 @deprecate conj(x) x
 
+# PR #26436
+@deprecate equalto(x) isequal(x)
+@deprecate(occursin(x), in(x))
+@deprecate_binding EqualTo Base.Fix2 false
+@deprecate_binding OccursIn Base.Fix2 false
```

Comment:
I left in definitions for these actually:

```
const EqualTo = Fix2{typeof(isequal)}
const OccursIn = Fix2{typeof(in)}
```

I'm not sure what their ultimate fate will be but they're pretty harmless for now.

Reply:
The use of `OccursIn` there will be kind of weird once the `contains` -> `occursin` replacement goes through though.

Reply:
Fair enough, but we have to do something. They can't both be deprecated just to `Fix2`, since that won't work.

Reply:
Okay, I can just remove these deprecations and we can probably just silently remove `EqualTo` and `OccursIn` between 0.7 and 1.0.

Reply:
Why not just do `@deprecate_binding EqualTo Base.Fix2{typeof(isequal)} false`?  That seems to work.

Reply:
Do we want to make the internal definitions use `<:Union{typeof(isequal), typeof(==)}`?